service/rating.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require_relative 'file_handler'
+
+# Description/Explanation of Rating module
+module Rating
+  def top_ids(n_times, orders, key_name)
+    orders.group_by { |object| object.instance_variable_get("@#{key_name}") }
+          .transform_values(&:count)
+          .sort_by { |_key, count| - count }[0...n_times]
+          .to_h
+          .keys
+  end
+
+  def top_books(_n_times, orders, books, top_ids)
```

Comment:
Why _n_times with _? And seems you don't use it in function, more attention

Reply:
Fixed